drakrun/drakrun/draksetup.py

```diff
@@ -41,14 +41,17 @@ def install(storage_backend, disk_size, iso_path, zfs_tank_name, max_vms, unatte
     os.makedirs(os.path.join(LIB_DIR, "profiles"), exist_ok=True)
     os.makedirs(os.path.join(LIB_DIR, "volumes"), exist_ok=True)
 
-    if os.path.exists("/etc/drakcore/config.ini"):
-        conf = configparser.ConfigParser()
-        conf.read(os.path.join(ETC_DIR, "config.ini"))
-        minio_access_key = conf.get('minio', 'access_key').strip()
+    conf = configparser.ConfigParser()
+    conf.read(os.path.join(ETC_DIR, "config.ini"))
+    minio_access_key = conf.get('minio', 'access_key').strip()
 
+    if os.path.exists("/etc/drakcore/config.ini"):
         if not minio_access_key:
             logging.info("Detected single-node setup, copying configuration from /etc/drakcore/config.ini")
```

Comment:
If we're using `ETC_DIR` variable, it would be nice to format message based on it

Reply:
`ETC_DIR` refers to the location of `/etc/drakrun`, not `/etc/drakcore`, so we would have to do something like `ETC_DIR + "../drakcore/config.ini"` which also doesn't look too elegant

I'm not sure whether the whole idea of `ETC_DIR` should be dropped